### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by jasonmacdonald** [typhonjs-node-tjsdoc/tjsdoc#7]
Like others, I just found my way here from ESdoc issues and I simply cannot comprehend how uncooperative that maintainer is. I'm so glad you "put your money where your mouth is" and actually forked/improved it! I can't wait to switch over!!!

Once I get my head wrapped around your new project I'll try to help out where I can. Great job! Please let me know if there are any areas you'd like specific help in.

Feel free to delete this, just wanted to say thanks and offer my help, if I can.

**Comment by typhonrt** [comment]
@jasonmacdonald welcome and thanks for the nod! 

Indeed the architecture differs somewhat from ESDoc and I'll definitely be creating an overview covering the details especially 3rd party plugin integration though that will be coming just before official launch, so there may be a bit of head scratching until these resources are available. 

An impetus for TJSDoc is to support the many module / repo vs mono repo approach. All other doc tooling only works well with mono repos. TJSDoc itself is highly modular and uses the eventbus pattern for inter-module communication via a plugin system and there are over 90 event bindings exposed in the core system accessible by any 1st / 3rd party plugin during the runtime which may be hard to follow currently. I'll be creating a plugin which documents this pattern along with interactive D3 graphs showing control flow of how modules are connected via events across modules which will significantly help in understanding how things work. IE by creating stellar documentation for TJSDoc itself the goal of supporting the many repo approach will be achieved, so a bit self-referential by design. 😄    

I did just post an update in the refactoring thread about current status, but right now I'm the bottleneck in getting the publisher module completed and checked in. 

I'll close this issue, but do follow #2 for ongoing updates. It'd be great to have you as an early user when the initial alpha release is ready as I'd like to do a round of wider testing before publicly announcing availability.
